cmd/infra/ibmcloud_powervs/validate.go

```diff
@@ -132,21 +132,25 @@ func validateCloudConnection(cloudConnName string, client *instance.IBMPICloudCo
 	}
 
 	if cloudConnL != nil {
+		if len(cloudConnL.CloudConnections) == 2 {
+			return "", fmt.Errorf("Two Cloud connections per cloud instance iss not supported currently")
+		}
+
```

Comment:
wondering if we can move this to createCloudConnection function only because lets say if user creates a custom dhcp network and cloud connection which is not creating a conflict then we can allow user to use that instead of always blocking based on this condition.

and also condition should don't allow to create cloud connection if there is even 1 existing in that zone per current design.

Reply:
@mkumatag I have a doubt here
In case user provides cloud connection, do we need to check whether that has 2 cloud connections or not?
Or do we need to assume that these conditions taken care and it will be going to work?

Reply:
It is good presume that everything is taken care by the user because that becomes out of band operation and it is always good to not check if user is supplying the value.